recipe/meta.yaml

```
@@ -10,40 +10,62 @@ source:
   sha256: bd94dd61edae3b23e43e70bbbfbada06e6d2f7fbac292ed1dbcfcb3afcca929a
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # because of restraint on pandas <2 missing for py312
+  skip: True  # [py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
     - setuptools-scm
+    - wheel
   run:
+    - python
     - importlib-metadata
-    - python >=3.6
-    - pandas
-    - numpy >=1.15.0
-    - statsmodels
+    - pandas <2
+    - numpy {{ numpy }}
```

```suggestion
    - numpy
```
As there is no `numpy` in `host`, and requirements say about a plain `numpy` https://github.com/jamalsenouci/causalimpact/blob/v0.2.6/requirements.in#L2

---

unfortunately the build fails without the pinning from cbc.